### ISSUE TITLE
Coded Domain and Subtypes not consistently showing up

### ISSUE THREAD

**Issue by dking159** [doug-m/CollectorAuroraTesting#115]
### Issue
On fields that are either subtypes or domains the value is showing up not the description.  This is inconsistent because some descriptions show up properly.  This is happening in the view and edit screens of a feature.  It also happens in related tables.

It might be related to numeric field types that have a domain.

Same map works in the current collector

### Map and Credentials

* Group Name:
* Map Name:

### Device Information
* Device:
* OS Version:

[ScreenRecording_06-18-2018 14-08-21.zip](https://github.com/doug-m/CollectorAuroraTesting/files/2112851/ScreenRecording_06-18-2018.14-08-21.zip)

**Comment by Kevin-GIS** [comment]
@dking159 when you collect a new feature, are the feature template names also displaying the code and not the description?

Can you share the map with me?

**Comment by dking159** [comment]
I don't think so.  The feature classes we are currently using are behind our firewall.  We can do a screen share or something like that.

________________________________

**Comment by Kevin-GIS** [comment]
Thanks @dking159 

A similar issue was logged, https://github.com/doug-m/CollectorAuroraTesting/issues/112, that has a map I can test with. 

I will reach out to you if I have further questions.

**Comment by Kevin-GIS** [comment]
Thanks @dking159 

I will try to reproduce this issue using my own data.

However if I am unable to reproduce this issue, would you be able to share a map package with me?

**Comment by dking159** [comment]
I'm sure we will be able to figure something out.


We also have a AGOL hosted feature service that has a similar issue.  The field was defined as 1-No and 0-Yes in the domains?


Here is the json for the field

  {
      "name" : "MAPCHANGESCOMPLETE",
      "type" : "esriFieldTypeSmallInteger",
      "alias" : "Map Changes Complete",
      "sqlType" : "sqlTypeOther",
      "nullable" : true,
      "editable" : true,
      "domain" :
      {
        "type" : "codedValue",
        "name" : "YES/NO",
        "codedValues" : [
          {
            "name" : "Yes",
            "code" : 0
          },
          {
            "name" : "No",
            "code" : 1
          }
        ]
      },
      "defaultValue" : null
    },


Dennis

________________________________

**Comment by Kevin-GIS** [comment]
Thanks for the information @dking159

**Comment by Kevin-GIS** [comment]
Reproduced the issue and logged SF 993
